tests/test_misc.py

```diff
@@ -0,0 +1,20 @@
+import warnings
+
+import numpy as np
+import pytest
+from conftest import assert_pint_equal
+
+from openscm.core.units import _unit_registry
+
+
+def test_pint_array_comparison():
```

Comment:
Shouldn't this go into `test_units.py`?

Reply:
No I want to put this function in `pint` eventually but just haven't got round to it yet so am leaving it here for now (it's not OpenSCM functionality, it's a function which should be in `pint` but isn't yet)

Reply:
Ah. Does it lead to missed lines in the test coverage in OpenSCM or why do you add it here?

Reply:
to make sure that the custom function `assert_pint_equal`, which is in `tests/conftest.py` does what I want it to